### ISSUE TITLE
Accessibility: Certain color combinations (text/background) do not have sufficient contrast

### ISSUE THREAD

**Issue by aphilop** [MERLTech/MERL-Center-public#83]
**Problem description**

There are two instances where this problem occurs:

1. Hero section in [Case Studies page](https://merlcenter.org/case-studies/)

![image](https://user-images.githubusercontent.com/102693686/168141947-25e9da68-1f7c-4e9c-a473-f0b7954e48f3.png)

Text color: #FFFFFF
Background color: #5ACAFA
Contrast ratio: 1.9:1

2. Link on hover
 
![LinkHover](https://user-images.githubusercontent.com/102693686/168147368-b7810fa9-2463-43d3-a406-e3872f2d4d01.png)

Text color: #073EF5 with .7 opacity, renders as #5178F8
Background color: #FFFFFF
Contrast ratio: 3.9:1

According to [W3C accessibility guidelines](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html) the contrast ratio should be at least 4.5:1

**Solution**
Define and use color combinations that meet the W3C minimum contrast requirement.

**Comment by aphilop** [comment]
Suggestions for resolving the above problems.

**Problem 1**
Use as background color the purple that is used in the MERL Center logo. 
Text color: #FFFFFF
Background color: #7A31F5
Contrast ratio: 5.9:1 (> 4.5:1)

![case-studies-hero](https://user-images.githubusercontent.com/102693686/168375243-53256f44-2974-4812-912c-2b6d5cb1debe.png)

The only problem with this approach is that the article list in the Home page will look like as in the image below: this purple is not very distinguishable from the blue used for the guides.

![article-list](https://user-images.githubusercontent.com/102693686/168375737-14afa9a4-6f43-43d1-8fa4-2bd31076d529.png)

**Problem 2**
Use a dotted underline for the link and a solid line when on hover (keep the blue color for both states).

![link-on-hover](https://user-images.githubusercontent.com/102693686/168376086-ca406756-3bea-48ad-bb7b-472375f9a549.png)
